pyproject.toml

```diff
@@ -5,9 +5,68 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools_scm]
 write_to = "waveorder/_version.py"
 
+[project]
+name = "waveorder"
+description = "Wave-optical simulations and deconvolution of optical properties"
+readme = "README.md"
+requires-python = ">=3.10"
+license = { file = "LICENSE" }
+authors = [{ name = "CZ Biohub SF", email = "compmicro@czbiohub.org" }]
```

Comment:
Change to match CITATION.cff?

Reply:
Whoops, missed this comment earlier. I think this is fine as is. No need to maintain two lists.